modules/core/quick/src/Traits/QuickQuantityTrait.php

```diff
@@ -42,6 +42,11 @@ public function createQuantity(array $values = []) {
       $term = $this->createOrLoadTerm($values['units'], 'unit');
       $values['units'] = $term;
     }
+    // Else check if a units term ID is provided and use that instead.
+    elseif (!empty($values['units_id'])) {
+      $values['units'] = $values['units_id'];
```

Comment:
I think `$values['units']` is expected to be an instance of `TermInterface` - but this would just make it a term ID integer, wouldn't it?

Reply:
Or is that also an acceptable input to `Quantity::create($values)`? Maybe there's some magic in there that loads the term automatically? Even so, I'd prefer to be consistent with our usage in this trait I think.

Reply:
Yea, I see what you are thinking. We could standardize these. In the end, the quantity unit field is an `EntityReferenceItem` to terms. It can accept a `TermInterface`, `['target_id' => $term_id]`, or because `target_id` is the main property id, just the `$term_id` is OK too.

What if we standardized on returning `['target_id' => $term_id]`, or even just the `$term_id` instead? That's a bit simpler, no reason to load the term if we don't need to. This method returns a quantity object which has helpers via the field methods for loading the term object if needed.